config/express.js

```diff
@@ -70,7 +70,7 @@ module.exports.http = {
       clientID: Reddit.data.clientID,
       clientSecret: Reddit.data.clientIDSecret,
       callbackURL: Reddit.data.redirectURL,
-      scope: "flair,modflair"
+      scope: "flair,modflair,modcontributors,wikiread,wikiedit,read,modposts"
```

Comment:
These are all the permissions that are needed.

* `modcontributors` to ban users
* `wikiread` and `wikiedit` to read and edit wiki pages (such as the automod config)
* `read` to use the reddit search function when removing the user's TSV threads
* `modposts` to remove TSV threads

We might consider having a separate mod-only permissions request, because other people might not be comfortable giving the app so many permissions.